doc/contributing/security-model-strategy.md

```diff
@@ -0,0 +1,77 @@
+# Security Model Strategy
+
+A clear security model, with features like permissions and policy enforcement,
+is a
+[top technical priority](https://github.com/nodejs/node/blob/master/doc/contributing/technical-priorities.md#permissionspoliciessecurity-model)
+of Node.js.
+
+## High-level approach
+
+* Document the security model
+* Document threat models and current state of the art
+* Support experimentation on features like permissions and policies
+* Add a security component in Node.js certification covering
+  the Node.js security model
+
+### Document the security model
+
+The current security model for Node.js is not yet well documented.
+At a high level it is:
+
+* Node.js does not provide a sandbox, both the JavaScript and
+  and Native code which is run is trusted to not be malicious.
+* The projects works to help code running on top of Node.js to avoid
```

Comment:
it has to be 'project works' or 'projects work'

Reply:
@marsonya thanks, fixed.